knox-webshell-ui/angular.json

```diff
@@ -54,18 +54,8 @@
           },
           "configurations": {
             "production": {
-              "budgets": [
-                {
-                  "type": "initial",
-                  "maximumWarning": "500kb",
-                  "maximumError": "1mb"
-                },
-                {
-                  "type": "anyComponentStyle",
-                  "maximumWarning": "2kb",
-                  "maximumError": "4kb"
-                }
-              ],
+              "buildOptimizer": false,
+              "aot": false,
```

Comment:
How does disabling ahead of time compilation affects the runtime performance?

Reply:
We have the same config in AdminUI and TokenManagementUI and there were no complaints so far about their performance. It's true that we may want AOT to be turned on, but I think it has to be in the scope of a different JIRA; right now fixing the build is more important.
What do you think?